### PR TITLE
Disable input and postproc by omiting config file sections

### PR DESCRIPTION
**Pull request by derekstavis**
I don't know if its the most elegant way of doing this, but it definitely works for me.

### CODE REVIEW

**Comment by KeyWeeUsr** [comment]
Seems good to me if it leaves the default behavior untouched (just an option to disable).

File: kivy/input/__init__.py [hunk 1]
```diff
@@ -23,14 +23,26 @@
 * Decreasing jittering
 * Decreasing the inaccuracy of touch on "bad" DIY hardware
 * Ignoring regions
+
+If your application don't use input or post processing you can remove the
```
Comment (KeyWeeUsr) [inline kivy/input/__init__.py]: Doesn't/won't/...

File: kivy/input/postproc/__init__.py [hunk 2]
```diff
@@ -13,12 +13,17 @@
 from kivy.input.postproc.retaintouch import InputPostprocRetainTouch
 from kivy.input.postproc.dejitter import InputPostprocDejitter
 from kivy.input.postproc.calibration import InputPostprocCalibration
+from kivy.config import Config
+
+# Enable postproc if the section exists in config file
+_postproc_enabled = (len(Config.items('input')) > 0 and
+                     len(Config.items('postproc')) > 0)
```
Comment (tshirtman) [inline kivy/input/postproc/__init__.py]: len(x) > 0 -> any(x)

**Comment by matham** [comment]
The Zen of python says `Explicit is better than implicit.`. With that change, removing a section implicitly disables it which I think is incorrect. There should be a way to explicitly disable them if they need to be disabled. Speaking of which, why do you want to disable it?

Also, they should be able to be disabled independently.

**Comment by tshirtman** [comment]
Closing until a rational for this behavior is provided, as merging this could certainly have surprising effects (double touch postproc anyone?).

**Pull request closed without merging**